src/pages/home/ReportScreen.js

```diff
@@ -275,7 +279,7 @@ class ReportScreen extends React.Component {
                                 >
                                     <HeaderView
                                         reportID={reportID}
-                                        onNavigationMenuButtonClicked={() => Navigation.navigate(ROUTES.HOME)}
+                                        onNavigationMenuButtonClicked={Navigation.goBack}
```

Comment:
> I think the back arrow on the report screen should take the user to LHN. @mountiny thought?

The Up action from report screen should take the user to the LHN on mobile, yes.

The back button should take the user to wherever they were before, it could have been a link to a report from somewhere else